### DISCUSSION TITLE
Customization possibilities of API overview

### DISCUSSION THREAD

**Discussion by jschaefer77** [stoplightio/elements#2043] [Q&A]
Hi, 
I would like to know if there are any possibilities to customize the API overview to:

1. Get more navigation entries on in the left column based on the headings in the API description. Not just overview.
2. Move version, base url, security and additional information section to the right column.

Both would make the UX in stoplight elements in my opinion much better.
Any thoughts, ideas on this?

regards Joerg

**Comment by manasesjesus** [comment]
I support this. ReDoc creates the navigation items based on the headers (h1, h2, h3) found in the`info.description` of the API, and this can be a markdown file. We use it like this:

```yml
openapi: 3.0.3
info:
  version: "Release 10.15.0"
  description:
    $ref: "markdown/reference-guide.md"
```

I saw that it was requested in #1128 since April 2021 and tagged as 'enhancement'...
Is this feature going to be included in Elements?
